openslides/motions/static/templates/motions/motion-list.html

```diff
@@ -77,12 +77,58 @@ <h1 translate>Motions</h1>
   </div>
 
   <div uib-collapse="!isSelectMode" class="row spacer">
-    <div class="col-sm-12 text-left">
+    <div class="col-sm-12 text-left form-inline" ng-show="isSelectMode" os-perms="motions.can_manage">
+      <!-- actions -->
+      <select ng-model="selectedAction" class="form-control">
+        <option value="" translate>--- Select action ---</option>
+        <option value="delete" translate>Delete</option>
+        <option value="setStatus" translate>Set status</option>
+        <option value="setCategory" ng-if="categories.length" translate>Set category</option>
+        <option value="setMotionBlock" ng-if="motionBlocks.length" translate>Set motion block</option>
+      </select>
+      <!-- state select -->
+      <select ng-show="selectedAction == 'setStatus'" ng-model="selectedState" class="form-control">
+        <option value="">--- <translate>Select state</translate> ---</option>
+        <option ng-repeat="state in states" ng-disabled="state.workflowHeader" value="{{ state.id }}">
+          {{ (state.workflowHeader ? state.headername : state.name) | translate }}
+        </option>
+      </select>
+      <!-- set state button -->
+      <a ng-show="selectedAction == 'setStatus' && selectedState"
+          ng-click="setStatusMultiple(selectedState)" class="btn btn-default">
+        <translate>Set status</translate>
+      </a>
+      <!-- category select -->
+      <select ng-show="selectedAction == 'setCategory'" ng-model="selectedCategory" class="form-control">
+        <option value="">--- <translate>Select category</translate> ---</option>
+        <option ng-repeat="category in categories" value="{{ category.id }}">
+          {{ category.name }}
+        </option>
+        <option value="none" translate>No category</option>
```

Comment:
Ah I see: Please rename this value `'none'` to something explicit like `'no_category_selected'`.